common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -9,6 +9,79 @@
 @elite_ranks = "gfx/interface/icons/culture_innovations/innovation_elite_ranks.dds"
```

Comment:
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/66#pullrequestreview-591275807

Reply:
![image](https://user-images.githubusercontent.com/74790101/108429919-9b90d180-720e-11eb-94d4-829e67a2df70.png)

In dev chat

Reply:
> ![image](https://user-images.githubusercontent.com/74790101/108429919-9b90d180-720e-11eb-94d4-829e67a2df70.png)
> 
> In dev chat

It's still not added.

Reply:
Its a different branch  https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/65